news/2019-04-08-project-loved-week-of-april-7th.md

```diff
@@ -0,0 +1,225 @@
+
+Our osu!mania friends interested in Project Loved are probably all aware of the unfortunate situation last round with Mat's Architecture voting (see [the last post on its forum thread](https://osu.ppy.sh/community/forums/posts/7031962) if you aren't caught up!). To help combat illegitimate votes, bandwagoning, and other related issues, we've implemented a measure to keep the voting results hidden until the polls are completed. We may look into other solutions if these problems occur again.
+
+In case you don't read the end of this post, be sure to submit suggestions for beatmaps to be added to Loved using the [Google form](https://docs.google.com/forms/d/e/1FAIpQLSdbgHOVqMF8wQQKSdddW1JhC10ff6C7fb4JbEW7PBQTn9gAqg/viewform)! All of the nominated beatmaps are chosen from submissions on this form, which you can view in the corresponding [Google sheet](https://docs.google.com/spreadsheets/d/1HgHwtO3kIzT8R4ocEJMZTosADrGJRJOFL-TZI97tZS4/edit). For osu!mania, please use [this Google form](https://docs.google.com/forms/d/e/1FAIpQLSeaGfoQNGMqw4qQcqRPItUZILh2fGwJR6ly6cZNY9OWPXkFhw/viewform) and [this Google sheet](https://docs.google.com/spreadsheets/d/1sjkTwUSvQ5Me-6rK61rToTg2bU-yX9X29CXdzttvhtM/edit) instead.
+
+### Navigation
+
+- [osu!](#osu)
+- [osu!taiko](#osutaiko)
+- [osu!catch](#osucatch)
+- [osu!mania](#osumania)
+
+## <a name="osu" id="osu"></a>osu!
+
+osu! Loved candidates were chosen by [DigitalHypno](https://osu.ppy.sh/users/4384207), [-Roxas](https://osu.ppy.sh/users/1986262), [BeasttrollMC](https://osu.ppy.sh/users/3171691), [Bubbleman](https://osu.ppy.sh/users/5182050) and [waywern2012](https://osu.ppy.sh/users/5870453)!
+
+---
+
+[![](/wiki/shared/news/2019-04-08-project-loved-week-of-april-7th/osu/hardware-store.jpg)](https://osu.ppy.sh/community/forums/topics/FORUM_TOPIC_ID)
+
+["Weird Al" Yankovic - Hardware Store](https://osu.ppy.sh/beatmapsets/833927#osu) by [Mr Moseby](https://osu.ppy.sh/users/3792472)  
+126 BPM, 3:30 | 7.75★, 8.93★, 9.01★  
+*written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*
+
+Over time we've seen quite a few competitive standard players dipping their toes into the mapping scene. One of these players is Woey (now known as Mr Moseby), a name recognizable not only for his skill but for his success in the mapping scene as well. Today we will be featuring Woey's take on "Weird Al" Yankovic's "Hardware Store", a map that has been notoriously popular amongst top players ever since its release. The mapset has three difficulties: "Matching Salt And Pepper Shakers" being the complete version at just over 9 stars, "POG moment" as a practice difficulty, and "a u t o m a t i c  c i r c u m c i s o r s (CS0)" for a CS0 version of the top difficulty. With the map featuring constant 252 BPM jumps, it's no surprise that the ones to popularize this map include some of the fastest players in the game.
```

Comment:
```suggestion
Over time we've seen quite a few competitive standard players dipping their toes into the mapping scene. One of these players is Woey (now known as Mr Moseby), a name recognizable not only for his skill, but for his success in the mapping scene as well. Today we will be featuring Woey's take on "Weird Al" Yankovic's "Hardware Store", a map that has been notoriously popular amongst top players ever since its release. This mapset has three difficulties: "Matching Salt And Pepper Shakers" being the complete version at just over 9 stars, "POG moment" as a practice difficulty, and "a u t o m a t i c  c i r c u m c i s o r s (CS0)" for a CS0 version of the top difficulty. With the map featuring constant 252 BPM jumps, it's no surprise that the ones to popularize this map include some of the fastest players in the game.
```